Lucidtech/Las/Client.cs

```diff
@@ -195,34 +243,33 @@ public Client(Credentials credentials)
         /// </example>
         /// <param name="documentId"> The document id to run inference and create a prediction on </param>
         /// <returns> Document information from REST API </returns>
-        public object GetDocument(string documentId)
-        {
+        public object GetDocument(string documentId) {
             RestRequest request = ClientRestRequest(Method.GET, $"/documents/{documentId}");
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
         /// <summary>
-        /// Post feedback to the REST API, calls the POST /documents/{documentId} endpoint.
-        /// Posting feedback means posting the ground truth data for the particular document.
+        /// Post ground truth to the REST API, calls the POST /documents/{documentId} endpoint.
+        /// Posting ground truth means posting the ground truth data for the particular document.
```

Comment:
Maybe 'post' is a bit misleading here, should we change it to 'Update ground truth for the document' or something similar?